lib/huginn_bigcommerce_product_agent/bigcommerce_product_agent.rb

```diff
@@ -103,22 +103,22 @@ def handle(event)
         def handle_variants(event)
             product = event.payload
 
-            wrapper_skus = {
-                physical: get_mapper(:ProductMapper).get_wrapper_sku_physical(product),
-                digital: get_mapper(:ProductMapper).get_wrapper_sku_digital(product),
-            }
-            bc_products = @product.get_by_skus(
-                wrapper_skus.map {|k,v| v},
-                %w[custom_fields options]
-            )
-
             split = get_mapper(:ProductMapper).split_digital_and_physical(
               product,
               interpolated['custom_fields_map']
             )
             physical = split[:physical]
             digital = split[:digital]
 
+            wrapper_skus = {
+                physical: get_mapper(:ProductMapper).get_wrapper_sku_physical(physical),
+                digital: get_mapper(:ProductMapper).get_wrapper_sku_digital(digital),
```

Comment:
@Jhalter5Stones You shouldn't need the `get_wrapper_sku_physical` and `get_wrapper_sku_digital` anymore. Since both of these functions are just calling `get_wrapper_sku` with a different subset of products, I would call that directly and remove these two overrides.